javascript/javascript3/week3/promises.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+let promise1 = fetch(
+  "https://api.github.com/search/repositories?q=user:cph-kiwi"
+);
+let promise2 = fetch(
+  " https://api.github.com/search/repositories?q=user:qamarfj"
+);
+let promise3 = fetch(
+  "https://api.github.com/search/repositories?q=user:sandhana14"
+);
+let promise4 = fetch(
+  "https://api.github.com/search/repositories?q=user:aijazraja"
+);
+let promise5 = fetch(
+  "https://api.github.com/search/repositories?q=user:sofiiadidovych"
+);
+let promise6 = fetch(
+  "https://api.github.com/search/repositories?q=user:Enqira"
+);
+let promise7 = fetch(
+  "https://api.github.com/search/repositories?q=user:TITHI1244"
```

Comment:
same API used 7 times. Better to use a constant like so: 
`const reposUrlUser = https://api.github.com/search/repositories?q=user:`